### ISSUE TITLE
Menu options very sensitive avoid scroll this menu

### ISSUE THREAD

**Issue by xzegga** [codrops/MultiLevelPushMenu#9]
I have problem in mobile device when I try to scroll this menu if ammount of menu options not fit on all height of the screen, Apears that the menu is very sensitive, when I try to scroll the options are activated and avoid to scroll the menu to view all options bellow.

Best regads

**Comment by FizzyGiant** [comment]
I've been having the same issue, I did you manage to find a fix?

**Comment by xzegga** [comment]
Yes, I resolve this issue, I don't remember at this time how to resolve but I will check my old projects to get solution for you.

**Comment by FizzyGiant** [comment]
I ended up getting around the issue by using Hammer.js and rewriting the javascript logic code to use functions provided by hammer.js instead.

I'd still be interested to see how you fixed it without using an external lib though, but don't worry if you can't find it :)

**Comment by Whetha** [comment]
Maybe this will help someone else....I figured out how to make the menu not activate the sub menu on a swipe. In the mlpushmenu.js change the event listener on line 136 to ignore swipes using jquery.mobile.  Add the parts in bold.
el.querySelector( 'a' ).addEventListener( self.eventtype, function( ev ) {
						**$("#mp-menu").on("swipe", function() {
							//do nothing on swipe
						});
						$("#mp-menu").on("click", function() {**
							ev.preventDefault();
							var level = closest( el, 'mp-level' ).getAttribute( 'data-level' );
							if( self.level <= level ) {
								ev.stopPropagation();
								classie.add( closest( el, 'mp-level' ), 'mp-level-overlay' );
								self._openMenu( subLevel );
							}
						**});**
					} );

**Comment by TonyShark19** [comment]
ev.preventDefault(); is preventing scrolling in the mlpushmenu.js

working on a fix for this....

**Comment by TonyShark19** [comment]
change line 136 to always be 'click'
el.querySelector( 'a' ).addEventListener('**click**', function( ev ) {

this was hanging up the menu on new page load so i added this to line 174
            //initialize menu closed
            self._resetMenu();

**Comment by jtphelan** [comment]
Changing line 136 worked for me, but adding self._resetMenu(); to line 174 gave me a "not a function error" in the console. Seems to work without it though. Not sure if the line numbers are off now, some context would help to know if I placed it in the right place.

**Comment by TonyShark19** [comment]
Glad the first part worked for you! I'm new to JS, so that's a win that it worked for both of us.
Here is the resetMenu() command in context. I added this because sometimes on page load the menu would load already opened. 
![image](https://cloud.githubusercontent.com/assets/13383517/9502817/12cce0b4-4bf9-11e5-8865-71c67d48a45b.png)

**Comment by jtphelan** [comment]
That works, thanks. I had it a couple lines below under _openMenu : function( subLevel ) which did not work.

**Comment by metew** [comment]
My solution/workaround.

.min file
{L="touchend";g="touchmove"} -> {L="click";g="touchmove"}

or if you want to read the full non min version; this is the offending line for scrolled content.
Replace the 'touchend' with 'click'
.js file (starts line 1109 for me)
            if( mobileCheck() ) {
                clickEventType = 'touchend';
                dragEventType = 'touchmove';
            }
